oaknational/Oak-Web-Application
# Feat/quiz style

## Description

styles added for all quiz types as described in #1185

## Issue(s)

Fixes #1185

## How to test

1. Go to 
/beta/teachers/key-stages/ks4/subjects/biology/units/cells-biology-3p8njnj/lessons/prokaryotic-and-eukaryotic-cells-b9qqye
beta/teachers/key-stages/ks3/subjects/science/units/atoms-and-the-periodic-table-9q9t0p6/lessons/development-of-the-periodic-table-r21gyc
3. Click on intro quiz and exit quiz
4. You should see styled quizzes

## Screenshots


How it should now look:
![image](https://user-images.githubusercontent.com/48293828/218432397-77e0ad62-7dfc-444a-b17c-21b6e76104f0.png)

## Checklist
